appveyor.yml

```diff
@@ -0,0 +1,66 @@
+version: 0.9.2.{build}
+image: Visual Studio 2017
+
+
+environment:
+  matrix:
+    - PlatformToolset: v140_xp
+
+platform:
+    - x64
+    - x86
+
+configuration:
+    - Release
+    - Debug
+
+install:
+    - if "%platform%"=="x64" set archi=amd64
+    - if "%platform%"=="x86" set archi=x86
+    - call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %archi%
+    - nuget restore "%APPVEYOR_BUILD_FOLDER%"\NppMenuSearch\packages.config -PackagesDirectory "%APPVEYOR_BUILD_FOLDER%"\packages
+
+
+build:
+    parallel: true                  # enable MSBuild parallel builds
+    verbosity: minimal
+
+build_script:
+    - cd "%APPVEYOR_BUILD_FOLDER%"
+    - msbuild NppMenuSearch.sln /m /p:configuration="%configuration%" /p:platform="%platform%" /p:PlatformToolset="%PlatformToolset%" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+
+after_build:
+    - cd "%APPVEYOR_BUILD_FOLDER%"\NppMenuSearch
+    - ps: >-
+
+        if ($env:CONFIGURATION -eq "Release") {
+            Push-AppveyorArtifact "bin\$env:PLATFORM\$env:CONFIGURATION\NppMenuSearch.dll" -FileName NppMenuSearch.dll
+        }
+
+        if ($($env:APPVEYOR_REPO_TAG) -eq "true" -and $env:CONFIGURATION -eq "Release" -and $env:PLATFORMTOOLSET -eq "v140_xp") {
+            if($env:PLATFORM -eq "x64"){
+            $ZipFileName = "NppMenuSearch_$($env:APPVEYOR_REPO_TAG_NAME)_x64.zip"
+            7z a $ZipFileName bin\$env:PLATFORM\$env:CONFIGURATION\NppMenuSearch.dll
+            }
+            if($env:PLATFORM -eq "x86"){
+            $ZipFileName = "NppMenuSearch_$($env:APPVEYOR_REPO_TAG_NAME)_x86.zip"
+            7z a $ZipFileName bin\$env:PLATFORM\$env:CONFIGURATION\NppMenuSearch.dll
+            }
+        }
+
+artifacts:
+  - path: NppMenuSearch_*.zip
+    name: releases
+
+deploy:
+    provider: GitHub
+    auth_token:
+        secure: !!TODO, see https://www.appveyor.com/docs/deployment/github/#provider-settings!!
```

Comment:
Dear @chcg,
sorry for the very long delay. I appreciate your commitment. I have no experience with CI systems and I have not enough time to learn it.
This line seems to suggest that I would need to configure my Github account settings? What is this "deploy:" section needed for? Would it be possible to not have this "deploy:" section and still let Appveyor supervise pull requests?

Reply:
@peter-frentrup The after_build, artifacts and deploy parts are optional. They are used to store the build results and make them accessible for testing and in case of deploy to automatically update github releases for tags with the build results from appveyor instead of building the binaries locally and upload them.
The deployment would require to grant appveyor access via a hashed github token.
So this part could be modified to your needs.